dashboard/app/assets/javascripts/videos.js.erb

```diff
@@ -143,7 +147,9 @@ function setupVideoFallback(videoInfo, playerWidth, playerHeight, shouldStillAdd
     if (!shouldStillAddCallback()) {
       return;
     }
-    addFallbackVideoPlayer(videoInfo, playerWidth, playerHeight);
+    setTimeout(function() {
```

Comment:
The setTimeout here duplicates the code seen on line 138; perhaps have the logic encapsulated in addFallbackVideoPlayer so that the caller doesn't have to worry about it?